doc_source/security-groups-for-pods.md

```diff
@@ -26,7 +26,8 @@ Before deploying security groups for pods, consider the following limits and con
       name: eks-vpc-resource-controller
   ```
 + If you're using [custom networking](cni-custom-network.md) and security groups for pods together, the security group specified by security groups for pods is used instead of the security group specified in the `ENIconfig`\.
-+ Pods using security groups must contain `terminationGracePeriodInSeconds` in their pod spec\. This is because the Amazon EKS VPC CNI plugin queries the API server to retrieve the pod IP address before deleting the pod network on the host\. Without this setting, the plugin doesn't remove the pod network on the host\. 
++ Pods using security groups must contain `terminationGracePeriodInSeconds` in their pod spec\. This is because the Amazon EKS VPC CNI plugin queries the API server to retrieve the pod IP address before deleting the pod network on the host\. Without this setting, the plugin doesn't remove the pod network on the host\.
++ Pods using Security Groups are not supported in clusters using [Nodelocal DNSCache](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/).
```

Comment:
Hi @frimgrandango , this _sounds_ right. How did you determine this? Were any pods talking via NodeLocal DNSCache didn't respect the SG associated with the pod?

Reply:
Hi @orsenthil. I reproduce the issue in my own environment for a customer when I was working in PS for AWS. 

The original issue was here:
https://github.com/aws/amazon-vpc-cni-k8s/issues/1384

It looks like M00nF1sh responded on an ETA for the FR:
https://github.com/aws/amazon-vpc-cni-k8s/issues/1662